about_asserts.exs

```diff
@@ -29,10 +29,10 @@ defmodule AboutAsserts do
     is_1_equal_2? = fn -> assert 1 == 2 end
     is_1_greater_than_2? = fn -> assert 1 > 2 end
 
-    message = "Assertion with " <> __? <> " failed"
+    message = "Assertion with #{__?} failed"
     assert_raise ExUnit.AssertionError, message, is_1_equal_2?
 
-    message = "Assertion with " <> __? <> " failed"
+    message = "Assertion with #{__?} failed"
```

Comment:
Honestly I'm biased to just leave it as is. The reader is welcome to decipher the syntax if they want. I do think though that we should introduce `<>` at some point. There's a note about that in https://github.com/iamvery/elixir-koans/issues/20.

Is that fair?